bin/commands/init/vsam/index.sh

```diff
@@ -59,105 +77,53 @@ if [ -z "${vsam_name}" ]; then
   print_error_and_exit "Error ZWEL0157E: Zowe Caching Service VSAM data set name (components.caching-service.storage.vsam.name) is not defined in Zowe YAML configuration file." "" 157
 fi
 
-jcl_existence=$(is_data_set_exists "${jcllib}(ZWECSVSM)")
-if [ "${jcl_existence}" = "true" ]; then
-  if [ "${ZWE_CLI_PARAMETER_ALLOW_OVERWRITE}" = "true" ]; then
-    # warning
-    print_message "Warning ZWEL0300W: ${jcllib}(ZWECSVSM) already exists. This data set member will be overwritten during configuration."
-  else
-    # print_error_and_exit "Error ZWEL0158E: ${jcllib}(ZWECSVSM) already exists." "" 158
-    # warning
-    print_message "Warning ZWEL0301W: ${jcllib}(ZWECSVSM) already exists and will not be overwritten. For upgrades, you must use --allow-overwrite."
-  fi
-fi
-
 # VSAM cache cannot be overwritten, must delete manually
 # FIXME: cat cannot be used to test VSAM data set
 vsam_existence=$(is_data_set_exists "${vsam_name}")
```

Comment:
`tso_is_data_set_exists` works with `VSAM`, returning `0` for success,

Reply:
do you mean `tso_is_data_set_exists` gives different behavior from `is_data_set_exists` ? I was encountering an issue with `is_data_set_exists` (would say false right after an ALLOCATE) so i wonder about that

Reply:
Yes, it `cat` vs `LIST DS`.